src/components/general/bitcoinApps/BitcoinApps.module.css

```diff
@@ -0,0 +1,39 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  background-image: url("../../../assets/general/bitcoinApps/background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 9.25rem max(3rem, calc(50vw - 32.8rem));
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.apps {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.appImage {
+  width: 6.25rem;
+  height: 6.25rem;
+}
+
+.appListWrapper {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  gap: 1.5rem;
+  align-items: flex-start;
+}
+
+.moreApps {
+  padding: 0.5rem 0;
+  cursor: pointer !important;
```

Comment:
nit: Would it be possible to do this without `important` i.e. by being more specific? On the other hand these are scoped styles, right?

Reply:
Fixed It 👍 
And, yes, those are scoped styles and won't affect anything other than its component.

Reply:
Uhm, I guess `NextLink` is somewhat of a default, right? Would it make sense to keep using that to keep everythign consistent? Mayeb it isn't worth switching that out just to get rid of a scoped `important`?

Reply:
Actually, there is a total of three cases currently implemented.
One is a simple `Link (native Next.js)` for use with text and icons.
The second one is where Button is supposed to redirect, so instead of styling `Link` as a button, we're using `Button (from Mantine) with the component as "a"`.
And the third case is where a Link is needed with the image but is not compatible with `Tooltip,` so we need to use `NextLink (from Mantine)`.

To simplify these, we can merge the use cases of `Link` and `NextLink` as `NextLink` would be superset for `Link.` So after that, there will be these cases: 
If a button redirects, then `Button with a` and for all other cases, `NextLink`?

Let me know what you think. I'll refactor the code accordingly.

Reply:
Sounds reasonable. Not need to block this though. Thanks for your explanation!

Reply:
Merge at will :)